iohub/cli/cli.py

```diff
@@ -87,3 +89,60 @@ def convert(input, output, grid_layout, chunks):
         chunks=chunks,
     )
     converter()
+
+
+@click.command()
```

Comment:
```suggestion
@cli.command()
```
This change will add it to the group so that you can type `iohub rename-wells`.